### PR TITLE
Fix code style

### PR DESCRIPTION
**Pull request by juliewww**
### Changed
Improved code style and readability across multiple files.
Updated test data for better alignment with the application's requirements.
### Fixed
Resolved issues with unnecessary code and lambda blocks.
Fixed string resources for better localization support.

### CODE REVIEW

**Comment by juliewww** [comment]
[@CodiumAI-Agent](https://github.com/CodiumAI-Agent)/describe

**Comment by juliewww** [comment]
[@CodiumAI-Agent](https://github.com/CodiumAI-Agent) /describe

File: core/ui/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -77,4 +77,5 @@
     <string name="core_ui_off" tools:override="true">Off</string>
     <string name="core_ui_collapse_list">Collapse list</string>
     <string name="core_ui_expand_list">Expand more</string>
+    <string name="core_ui_processing">Processing</string>
```
Comment (lihenggui) [inline core/ui/src/main/res/values/strings.xml:80]: I think the original content description is `loading` not `processing`

**Comment by juliewww** [comment]
@CodiumAI-Agent /update_changelog

**Review by lihenggui — APPROVED** [review]

**Pull request merged**